server/app/services/program/predicate/Operator.java

```diff
@@ -16,8 +16,8 @@ public enum Operator {
       "is greater than or equal to",
       ImmutableSet.of(ScalarType.CURRENCY_CENTS, ScalarType.LONG)),
   IN("in", "is one of", ImmutableSet.of(ScalarType.STRING, ScalarType.LONG)),
-  IS_AFTER(">=", "is later than", ImmutableSet.of(ScalarType.DATE)),
-  IS_BEFORE("<=", "is earlier than", ImmutableSet.of(ScalarType.DATE)),
+  IS_AFTER(">", "is later than", ImmutableSet.of(ScalarType.DATE)),
```

Comment:
I'm thinking IS_AFTER maybe should still be >=  as conceptually date comparisons probably want "on or after" and "before".  or maybe we need a third one to handle that?

Might we worth asking @bion  if there's backing UX research on this

Reply:
+1 to Shane's interpretation

Reply:
Let's make sure to sync with Elise and Bloomington folks on the removal of = from IS_AFTER

This will change any predicate evaluation they have on dates, though they may not have any.